src/unit_tests/wazuh_db/test_wdb.c

```diff
@@ -279,32 +279,84 @@ void test_wdb_exec_row_stmt_one_text(void **state) {
     cJSON_Delete(result);
 }
 
-void test_wdb_exec_row_stmt_done(void **state) {
+void test_wdb_exec_row_stmt_multi_column_done(void **state) {
     test_struct_t *data  = (test_struct_t *)*state;
 
     expect_sqlite3_step_call(SQLITE_DONE);
 
     int status = 0;
-    cJSON* result = wdb_exec_row_stmt(*data->wdb->stmt, &status);assert_null(result);
+    cJSON* result = wdb_exec_row_stmt_multi_column(*data->wdb->stmt, &status);
+
+    assert_null(result);
     assert_int_equal(status, SQLITE_DONE);
     assert_null(result);
 }
 
-void test_wdb_exec_row_stmt_error(void **state) {
+void test_wdb_exec_row_stmt_multi_column_error(void **state) {
     test_struct_t *data  = (test_struct_t *)*state;
 
     expect_sqlite3_step_call(SQLITE_ERROR);
     expect_string(__wrap__mdebug1, formatted_msg, "SQL statement execution failed");
 
     int status = 0;
-    cJSON* result = wdb_exec_row_stmt(*data->wdb->stmt, &status);
+    cJSON* result = wdb_exec_row_stmt_multi_column(*data->wdb->stmt, &status);
     assert_int_equal(status, SQLITE_ERROR);
     assert_null(result);
 }
 
 /* Tests wdb_exec_stmt_sized */
 
-void test_wdb_exec_stmt_sized_success(void **state) {
+void test_wdb_exec_stmt_sized_success_single_column_string(void **state){
+    test_struct_t *data  = (test_struct_t *)*state;
+    char col_text[4][16] = { 0 };
+    int status = SQLITE_ERROR;
+
+    for (int i = 0; i < 4; ++i) {
+        expect_sqlite3_step_call(SQLITE_ROW);
+        will_return(__wrap_sqlite3_column_count, 1);
+        expect_value(__wrap_sqlite3_column_type, i, 0);
+        will_return(__wrap_sqlite3_column_type, SQLITE_TEXT);
+        snprintf(col_text[i], 16, "COL_TEXT_%d", i);
+        expect_value(__wrap_sqlite3_column_text, iCol, 0);
+        will_return(__wrap_sqlite3_column_text, col_text[i]);
+    }
+
+    expect_sqlite3_step_call(SQLITE_DONE);
+
+    cJSON* result = wdb_exec_stmt_sized(*data->wdb->stmt, WDB_MAX_RESPONSE_SIZE, &status, STMT_SINGLE_COLUMN);
+    char *ret_str = cJSON_PrintUnformatted(result);
```

Comment:
It is a minor thing, but should we define a unique way to declare pointer variables? i.e. type+*+space+variable_name